### PR TITLE
#17: Fixed NullPointerException in unquotedParameter

### LINKED ISSUE #17: nullPointerException in unquotedParameter
I think I already reported this problem, but I encountered it again: `nullPointerException` when the argument of unquotedParameter is null. Please validate the argument

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #17

### CODE REVIEW

File: doc/changes/changes_0.2.2.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# error-reporting-java 0.2.2, released 2020-11-30
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_0.2.2.md]: Update the date or use the latest RR for releasing (I think I sent you jar)

Comment (AnastasiiaSergienko) [reply doc/changes/changes_0.2.2.md]: But it will be an additional commit on main

File: doc/changes/changes_0.2.2.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Code Name: Fixed null value for unquoted parameters
+
+## Bugfixes
+
+* # 17: Fixed NullPointerException in unquotedParameter
+
+# Dependency Updates:
+
+* Updated `com.exasol:project-keeper-maven-plugin:0.3.0` to 0.4.2
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_0.2.2.md]:
```suggestion
* Updated `com.exasol:project-keeper-maven-plugin:0.3.0` to `0.4.2`
```

**Review by AnastasiiaSergienko — APPROVED** [review]
Approved with editorial change

File: doc/changes/changes_0.2.2.md (hunk 1 above)
Comment (jakobbraun) [reply doc/changes/changes_0.2.2.md]:
```suggestion
# error-reporting-java 0.2.2, released 2020-12-17
```

**Pull request merged**